### PR TITLE
Add game sales/speedrun tracking

### PR DESCRIPTION
**Pull request by SukrutN**
Add game sales/speedrun tracking

### CODE REVIEW

File: src/app/Leaderboards/page.tsx [hunk 1]
```diff
@@ -1,72 +1,40 @@
-import React from 'react';
+import React from "react";
 
 const Leaderboards = () => {
-  // Example leaderboard data
-  const leaderboardData = [
```
Comment (ThatZiv) [inline src/app/Leaderboards/page.tsx:5]: Good that we removed this static data

File: src/app/Leaderboards/page.tsx [hunk 2]
```diff
@@ -1,72 +1,40 @@
...
+          <div className="col-span-1 flex justify-center sm:col-span-2 lg:col-span-3">
+            <div className="rounded-xl bg-white p-10 shadow-xl transition-transform duration-300 ease-in-out hover:scale-110 dark:bg-gray-700">
+              <h2 className="mb-6 text-center text-4xl font-semibold text-gray-800 dark:text-gray-100"></h2>
+              <p className="text-xl text-gray-600 dark:text-gray-300"></p>
+              <div
+                className="flex items-center justify-center overflow-hidden rounded-xl bg-gray-200 dark:bg-gray-600"
+                style={{ height: "1000px", width: "800px" }}
+              >
+                <iframe
+                  src="https://gg.deals/"
```
Comment (ThatZiv) [inline src/app/Leaderboards/page.tsx:26]: Do we have a failsafe when gg.deals is down?

File: src/app/Speedrunning/page.tsx [hunk 3]
```diff
@@ -1,81 +1,42 @@
...
+              <h2 className="mb-6 text-center text-4xl font-semibold text-gray-800 dark:text-gray-100"></h2>
+              <p className="text-xl text-gray-600 dark:text-gray-200">
+                Check out the fastest times in any game!
+              </p>
+              <div
+                className="mt-10 flex h-64 items-center justify-center overflow-hidden rounded-xl bg-gray-200 dark:bg-gray-600"
+                style={{ height: "1000px", width: "800px" }}
+              >
+                <iframe
+                  src="https://www.speedrun.com/sm64"
```
Comment (ThatZiv) [inline src/app/Speedrunning/page.tsx:28]: Do we have a failsafe when speedrun.com is down?

File: src/comps/Nav.tsx [hunk 4]
```diff
@@ -171,14 +171,6 @@ export default function Nav() {
         >
           Upcoming
         </Navbar.Link>
-
-        <Navbar.Link
-          as={Link}
-          href="/HowTtWorks"
-          active={pathname.startsWith("/HowTtWorks")}
```
Comment (ThatZiv) [inline src/comps/Nav.tsx:178]: Path directories traditionally are not case-sensative and are lower-case. Please follow conventions next time

**Review by ThatZiv — APPROVED** [review]
Looks good to me!

File: src/app/Leaderboards/page.tsx [hunk 5]
```diff
@@ -1,72 +1,40 @@
-import React from 'react';
```
Comment (PierreT20) [inline src/app/Leaderboards/page.tsx:1]: Solid import of the same thing

File: src/app/Leaderboards/page.tsx [hunk 6]
```diff
@@ -1,72 +1,40 @@
...
-    // More entries can be added here
-  ];
-
   return (
-    <section id="leaderboards" className="bg-gray-100 py-12 dark:bg-gray-800">
-      <div className="container mx-auto px-4">
-        <h1 className="mb-6 text-4xl font-bold text-center text-gray-800 dark:text-white">
-          Leaderboards
+    <section
+      id="charts"
```
Comment (PierreT20) [inline src/app/Leaderboards/page.tsx:6]: nice styling

File: src/app/Leaderboards/page.tsx [hunk 7]
```diff
@@ -1,72 +1,40 @@
...
-                    {entry.score.toLocaleString()} {/* Formatting large numbers */}
-                  </td>
-                  <td className="px-6 py-4 whitespace-nowrap text-sm text-gray-500 dark:text-gray-200">
-                    {entry.game}
-                  </td>
-                </tr>
-              ))}
-            </tbody>
-          </table>
+        <div className="grid grid-cols-1 gap-10 sm:grid-cols-2 lg:grid-cols-3 xl:gap-x-14">
```
Comment (PierreT20) [inline src/app/Leaderboards/page.tsx:16]: more nice styling

File: src/app/Speedrunning/page.tsx [hunk 8]
```diff
@@ -1,81 +1,42 @@
...
   return (
-    <section id="speedrunning" className="bg-gray-100 py-12 dark:bg-gray-800">
-      <div className="container mx-auto px-4">
-        <h1 className="mb-6 text-4xl font-bold text-center text-gray-800 dark:text-white">
-          Speedrunning Records
+    <section
+      id="charts"
+      className="bg-gray-50 py-12 transition duration-500 ease-in-out dark:bg-gray-900"
+    >
+      <div className="container mx-auto max-w-7xl px-4 sm:px-6 lg:px-8">
```
Comment (PierreT20) [inline src/app/Speedrunning/page.tsx:9]: LGTM

**Review by PierreT20 — APPROVED** [review]

File: src/app/about/page.tsx [hunk 9]
```diff
@@ -13,7 +13,7 @@ export const metadata: Metadata = {
 const AboutPage = () => {
   return (
     <main>
-      <Breadcrumb pageName="About Us Page" />
+      <Breadcrumb pageName="Community Chatroom" />
```
Comment (XLRA) [inline src/app/about/page.tsx:16]: Very nice addition!

File: src/components/Features/featuresData.test.tsx [hunk 10]
```diff
@@ -1,40 +1,40 @@
...
+      expect(typeof feature.title).toBe("string");
+
+      expect(feature).toHaveProperty("paragraph");
+      expect(typeof feature.paragraph).toBe("string");
+
+      expect(feature).toHaveProperty("btn");
+      expect(typeof feature.btn).toBe("string");
+
+      expect(feature).toHaveProperty("btnLink");
+      expect(typeof feature.btnLink).toBe("string");
```
Comment (XLRA) [inline src/components/Features/featuresData.test.tsx:30]: Good test changes!

**Review by XLRA — APPROVED** [review]
LGTM!

**Pull request merged**